src/main/kotlin/com/terraformation/backend/customer/model/IndividualUser.kt

```diff
@@ -130,39 +108,24 @@ data class IndividualUser(
   private val _globalRoles = ResettableLazy { permissionStore.fetchGlobalRoles(userId) }
   override val globalRoles: Set<GlobalRole> by _globalRoles
 
-  /** History of permission checks performed in the current request or job. */
-  val permissionChecks: MutableList<PermissionCheck> = mutableListOf()
-
   override fun clearCachedPermissions() {
     _organizationRoles.reset()
     _facilityRoles.reset()
     _globalRoles.reset()
   }
 
   val fullName: String?
-    get() = makeFullName(firstName, lastName)
-
-  override fun <T> run(func: () -> T): T {
-    return CurrentUserHolder.runAs(this, func, authorities)
-  }
+    get() = TerrawareUser.makeFullName(firstName, lastName)
 
   /** Returns true if the user is an admin, owner or Terraformation Contact of any organizations. */
   override fun hasAnyAdminRole() =
       organizationRoles.values.any {
         it == Role.Owner || it == Role.Admin || it == Role.TerraformationContact
       }
 
-  override fun isAdminOrHigher(organizationId: OrganizationId?): Boolean {
-    return organizationId?.let {
-      recordPermissionCheck(RolePermissionCheck(Role.Admin, organizationId))
-      when (organizationRoles[organizationId]) {
-        Role.Admin,
-        Role.Owner,
-        Role.TerraformationContact -> true
-        else -> false
-      }
-    } ?: false
-  }
+  override fun getName(): String = authId ?: throw IllegalStateException("User is unregistered")
+
+  override fun getUsername(): String = authId ?: throw IllegalStateException("User is unregistered")
```

Comment:
These would probably make sense to move to `TerrawareUser` since they're the same for funder users and device managers (though not for the system user, so they can stay overridden in `SystemUser`.)

Reply:
Ya I guess the DeviceManager user will always have its `authId`. I didn't move it there originally because the implementation above throws an error if it doesn't exist instead of just returning null